README.md

```diff
@@ -118,4 +118,38 @@ VSCode supports running solargraph through an [extension](https://marketplace.vi
 
 #### Sublime Text
 
-...
+[LSP for Sublime Text](https://lsp.sublimetext.io/) can be configured to run Solargraph.
+
+- Install the LSP package using the command palette (`Tools > Command Palette`):
+  - Enter the command `Package Control: Install Package`
+  - Choose the LSP package
+- Add Solargraph to the list of LSP clients in the package's settings (`Settings > Package Settings > LSP > Settings):
+
+```json
+{
+  "clients": {
+    "solargraph": {
+      "enabled": true,
+      "command": [
+        "solargraph", "stdio"
+      ],
+      "selector": "source.ruby",
+      "initializationOptions": {
+        "diagnostics": true
+      }
+    }
+  }
+}
+```
+
+The LSP package supports automatic formatting but doesn't enable it by default. If you'd like to use it:
```

Comment:
There is one small wrinkle to using this feature, but it isn't specific to Sublime Text so this felt like the wrong spot to add it: Solargraph uses Rubocop for formatting (I think, anyway?) and if you're running a newer version of Ruby then Rubocop's formatter will make changes that don't work with DragonRuby's version of mruby. The workaround I've been using is adding these lines to my `.rubocop.yml` file:

```yaml
AllCops:
  TargetRubyVersion: 2.7
  
Style/RandomWithOffset:
  Enabled: false
```

I can add that somewhere else in the README, but is it getting too much into Solargraph config details that people should go to their docs for?

Reply:
I'm inclined to leave it out for now - lets see if anyone else gets snagged by it.

My gut tells me that someone who enables this is proficient enough in Rubocop to know how to do it.

I appreciate the attention to detail though, thanks @bradJM

Reply:
Sounds good!